dev-resources/Makefile.i18n

```diff
@@ -0,0 +1,123 @@
+# -*- Makefile -*-
+# This file was generated by the i18n leiningen plugin
+# It currently has a typo in autogeneration ">&" needs to be "&>"
+# Change your auto-generated makefile to look like this one 
+# There is a PR open at the moment, see for more details: https://github.com/puppetlabs/clj-i18n/pull/22 
```

Comment:
Unfortunately no. What happens is: 
- i18 gets initialized
- current makefile in directory gets overwritten, points to i18n makefile